src/components/Table.tsx

```diff
@@ -1031,9 +1031,9 @@ const $Tbody = styled.tbody<TableStyleProps>`
     `}
 `;
 
-const $Row = styled.div<{ uiRefreshEnabled: boolean }>`
+const $Row = styled.div<{ $uiRefreshEnabled: boolean }>`
```

Comment:
At this point I wonder if we should have added it to the global styled components context thing so everything has access to this variable w/ no added plumbing.

Reply:
oh that would've been nice 🤔 HAHAH maybe i will go back and refactor, maybe it's just a lesson for next time